addonscript/task.py

```diff
@@ -0,0 +1,40 @@
+"""Methods that deal with the task metadata to supply addonscript."""
+
+import os
+from scriptworker.exceptions import TaskVerificationError
+
+
+def get_channel():  # noqa
+    raise Exception()
+
+
+def build_filelist(context):
```

Comment:
There are a number of helper functions in `scriptworker.artifacts` that do this -- do they work here?
https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/artifacts.py#L321

Reply:
Doesn't look like any do this for me....